.travis.yml

```diff
@@ -1,27 +1,85 @@
 language: cpp
 dist: trusty
-sudo: required # the doc target uses sudo to install dependencies
+sudo: false
 
-os:
-  - linux
-  - osx
+os: linux
+
+git:
+  depth: 1
+
+addons:
+  apt:
+    update: true
+    sources: &apt_sources
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-6
 
 env:
   global:
     - secure: |-
         a1eovNn4uol9won7ghr67eD3/59oeESN+G9bWE+ecI1V6yRseG9whniGhIpC/YfMW/Qz5I
         5sxSmFjaw9bxCISNwUIrL1O5x2AmRYTnFcXk4dFsUvlZg+WeF/aKyBYCNRM8C2ndbBmtAO
         o1F2EwFbiso0EmtzhAPs19ujiVxkLn4=
-  matrix:
-    - BUILD=Doc
-    - BUILD=Debug STANDARD=14
-    - BUILD=Release STANDARD=14
 
 matrix:
   exclude:
-    - os: osx
-      env: BUILD=Doc
+    - env: TRAVIS_EMPTY_JOB_WORKAROUND=true
```

Comment:
What is this for?

Reply:
When creating a Travis build matrix, that is sometimes necessary. I just forgot it there.